addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -154,7 +160,9 @@ $o-we-zindex: $o-we-overlay-zindex + 1 !default;
 
 // Toolbar - Generic rules shared among backend & frontend
 .oe-toolbar {
-    display: grid;
+
```

Comment:
@stefanorigano knows the guidelines better than I do but I believe this line is superfluous?